auth-web/src/assets/scss/theme.scss

```diff
@@ -1,6 +1,7 @@
 // Gray Palette
 $gray0: #f8f9fa;
 $gray1: #f1f3f5;
+$grat1B: #f2f2f2; // BC GOV BACKGROUND
```

Comment:
Typo? Also, "1B" isn't descriptive. How about calling is BcGovBg or something like that?

Note that in other places we've just specified a colour class of "gray lighten-3". Would that work here or were you told to use this exact gray?

Reply:
oops, type for sure 👍 
There is a new style guide (in the latest Invision designs from Tracey) which specifies this colour exactly. I like the idea of the more specific name tho, will update this!

Reply:
Updated in next commit 👍